hedyorg/hedy
# 🪲 Fix stop program being shown indefinitely

The problem was that we were trying to use the `theGlobalDebugger` variable while it was undefined. By using the `?` operator of typescript we guarantee that we won't be trying to call a method if the object is null.

Fixes #5555

**How to test**

I added a Cypress test., but also you can try running programs and see if the button works normally again!
